### DISCUSSION TITLE
Please someone explain this to me `export const fetchCache = 'force-no-store';`.

### DISCUSSION THREAD

**Discussion by abhishekmardiya** [vercel/next.js#65778] [App Router] [answered]
`export const fetchCache = 'force-no-store';
`

**Comment by MangoMarcus** [comment, marked as answer]
You can see the documentation for this here: https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#fetchcache

Using `force-no-store` apparently overrides all `fetch` requests to use `cache: 'no-store'`, regardless of what you pass
